bin/probot-receive.js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require("../lib/bin/probot-receive");
+import "../lib/bin/probot-receive.js";
```

Comment:
What if we just do
```suggestion
import("../lib/bin/probot-receive.js");
```
In the bin files and already apply them into beta?

Shouldnt be breaking and reducing the esm switch effort.

Reply:
Sounds good, I'll apply this on the other branch later.